openmc/settings.py

```diff
@@ -125,6 +126,8 @@ class SettingsFile(object):
         Coordinates of the lower-left point of the UFS mesh
     ufs_upper_right : tuple or list
         Coordinates of the upper-right point of the UFS mesh
+    resonance_scattering : ResonanceScattering or iterable thereof
```

Comment:
That will make it over 80 columns.  Do I need to do anything special to show that the first line in that comment has been continued?
```python
    """
    resonance_scattering : ResonanceScattering or iterable of
        ResonanceScattering
        The elastic scattering model to use for resonant isotopes.
    """
```